### PR TITLE
LPS-144065 Update the 'getAttribute' method at some tests and macro.

### PR DESCRIPTION
**Pull request by mateusralv**
Ticket: 
[LPS-144065](https://issues.liferay.com/browse/LPS-144065)

**Test fixed:** 
EditClickElementTarget;

**Macro Fixed:** 
takeElementID.

### CODE REVIEW

**Comment by mateusralv** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase [hunk 1]
```diff
@@ -258,10 +258,6 @@ definition {
 				testTitle = "AB test title");
 		}
 
-		task ("Take the ID Element") {
-			var idbutton = ABTest.takeElementID();
-		}
-
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase:264]: This steps was duplicate in the test.

**Comment by mateusralv** [comment]
Hey @Tim-Cao, could you please review this PR? Thank you.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase [hunk 2]
```diff
@@ -1530,7 +1526,7 @@ definition {
 				key_index = "1",
 				locator1 = "ABTest#SELECT_CLICKABLE_ELEMENT");
 
-			var idelement = selenium.getAttribute(locator1 = "//div//div[contains(@class,'text-truncate')]@title");
+			var idelement = selenium.getAttribute("//div//div[contains(@class,'text-truncate')]@title");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase:1529]:
```suggestion
			var idelement = selenium.getAttribute("//div[contains(@class,'text-truncate')]@title");
```

**Review by Tim-Cao — COMMENTED** [review]
Hi @mateusralv, Could you update according to comments? Thanks

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by mateusralv** [comment]
Hey @Tim-Cao, could you please review my updates? Thank you.

**Comment by Tim-Cao** [comment]
LGTM! Good job @mateusralv

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**